miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/form/PaneTag.java

```diff
@@ -43,7 +43,7 @@ protected int doStartTagInternal() throws Exception {
     ObjectMapper mapper = new ObjectMapper();
 
     pageContext.getOut().append(String.format(
-        "<div id='%1$s'></div><script type='text/javascript'>jQuery(document).ready(function () { Tile.createPane('%1$s', TileTarget.%2$s, %3$s);});</script>",
+        "<div id='%1$s'></div><script type='text/javascript'>jQuery(document).ready(function () { Pane.createPane('%1$s', PaneTarget.%2$s, %3$s);});</script>",
```

Comment:
Since `Pane.createPane` has been moved down into the given `PaneTarget`, `PaneTag` will require a new `PaneTarget` to be written in order to be used.  This make sense, since we would likely have to write a new `PaneTarget` if we added a widget that used `PaneTag`, anyway.

Reply:
Note: this class is not currently used, but will likely be handy in the future.